tests/ProductBundle/Seo/Services/TwitterTest.php

```diff
@@ -46,18 +68,71 @@ public function testAlterPage(): void
         $currencyDetector = $this->createMock(CurrencyDetectorInterface::class);
         $product = new ProductTwitterMock();
 
+        $numberHelper->expects($this->any())
+                ->method('formatCurrency')
+                ->willReturn($product->getPrice());
+
+        $site = 'mySiteName';
+        $creator = 'Sonata';
+
         //Prepare currency
         $currency = new Currency();
         $currency->setLabel('EUR');
         $currencyDetector->expects($this->any())
                 ->method('getCurrency')
                 ->willReturn($currency);
 
-        $twitterService = new Twitter($mediaPool, $numberHelper, $currencyDetector, 'test', 'test', 'test', 'test', 'reference');
+        $twitterService = new Twitter($mediaPool, $numberHelper, $currencyDetector, $site, $creator, 'test', 'test', 'reference');
         $twitterService->alterPage($seoPage, $product);
         $content = $extension->getMetadatas();
 
-        $this->assertContains('twitter:label1', $content);
+        $this->assertContains('name="twitter:label1" content="Price"', $content);
         $this->assertNotContains('twitter:image:src', $content);
+        $this->assertContains('name="twitter:site" content="mySiteName"', $content);
+        $this->assertContains('name="twitter:creator" content="Sonata"', $content);
+        $this->assertContains('name="twitter:data2" content="Sku1"', $content);
+        $this->assertContains('name="twitter:title" content="Product1"', $content);
+        $this->assertContains('name="twitter:description" content="O-some product"', $content);
+        $this->assertContains('name="twitter:data1" content="123.56"', $content);
+    }
+
+    public function testAlterPageImage(): void
+    {
+        $mediaPool = $this->createMock(Pool::class);
+        $seoPage = new SeoPage('test');
+        $extension = new SeoExtension($seoPage, 'UTF-8');
+        $numberHelper = $this->createMock(NumberHelper::class);
+        $currencyDetector = $this->createMock(CurrencyDetectorInterface::class);
+        $product = new ProductTwitterMock();
+
+        //Prepare currency
+        $currency = new Currency();
+        $currency->setLabel('EUR');
+        $currencyDetector->expects($this->any())
+                ->method('getCurrency')
+                ->willReturn($currency);
+
+        $site = 'mySiteName';
```

Comment:
This are OTV (one time variables). You should avoid these.

After fixing this, it's RTM

Reply:
Rtm ?

Reply:
Ready To Merge / Release To Manufacturer

https://www.urbandictionary.com/define.php?term=RTM